certified-connectors/DocuSignDemo/script.csx

```diff
@@ -1631,30 +1631,20 @@ public class Script : ScriptBase
       var query = HttpUtility.ParseQueryString(this.Context.Request.RequestUri.Query);
 
       var newSigner = new JObject();
+      newSigner["signer"] = null;
+      var newBody = new JObject();
       var recipientEmailId = query.Get("recipientEmail");
       var phoneNumber = query.Get("areaCode") + " " + query.Get("phoneNumber");
       var signerPhoneNumber = "";
       char[] charsToTrimPhoneNumber = { '*', ' ', '\'', '/', '-', '(', ')', '+'};
 
-      if ((recipientEmailId == null) && (query.Get("phoneNumber") == null))
```

Comment:
keep this exception checking code here. because you are using recipientEmailId and phoneNumber below.

Reply:
you are right. I do see you are checking above it.